beacon/blockchain/deposit.go

```diff
@@ -33,6 +33,7 @@ import (
 // defaultRetryInterval processes a deposit event.
 const defaultRetryInterval = 20 * time.Second
 
+// depositFetcher fetches the blocks at blockNum-s.eth1FollowDistance so that they can be included in blockNum+1.
```

Comment:
nit: can you just add here that this func ALSO writes these deposits to deposit store